specification/MDF.tex

```diff
@@ -280,24 +280,45 @@ \subsection{Acquisition Parameters (group: \inlvar{/acquisition/}, non-optional)
 \begin{multicols}{2}
 \paragraph{Remarks:} The acquisition parameter group can describe different imaging protocols and trajectory settings. The corresponding data is organized into general information, a subgroup containing information on the $D$ excitation channels, and a subgroup containing information on the $C$ receive channels.
 
-In MPI, a frame groups all data used to reconstruct a single MPI image/tomogram. In the simplest scenario, this data is acquired during one \inlvar{/drivefield/period}. The acquisition time increases by a factor of \inlvar{numAverages} if averaging is applied and by a factor of \inlvar{numPeriods} if data is recorded over several (averaged) cycles. In a multi-patch setting $J$ \inlvar{offsetField}s or mechanical movements shift the gradient field by \inlvar{offsetFieldShift}, where $J$ is the number of patches \inlvar{numPatches} of a multi-patch measurement. For each shift at least one full drive field cycle is used to acquire the measurement data. For instance, a Cartesian 2D trajectory with 100 lines could be realized by setting \mbox{\inlvar{numPatches} $ = 100$}.
+In MPI, a frame groups all data used to reconstruct a single MPI image/tomogram. In the simplest scenario, this data is acquired during one \inlvar{/drivefield/period}. The acquisition time increases by a factor of \inlvar{numAverages} if averaging is applied and by a factor of \inlvar{numPeriods} if data is recorded over several (averaged) cycles. 
+%In a multi-patch setting $J$ \inlvar{offsetField}s or mechanical movements shift the gradient field by \inlvar{offsetFieldShift}, where $J$ is the number of patches \inlvar{numPatches} of a multi-patch measurement. For each shift at least one full drive field cycle is used to acquire the measurement data. %For instance, a Cartesian 2D trajectory with 100 lines could be realized by setting \mbox{\inlvar{numPatches} $ = 100$}.
+
+\textcolor{red}{TODO: Discuss the dimension $Y$. It is usually equal to $J$ when considering a static multi-patch scenario. But it can also be finer resolved to describe continuous FF.}
```

Comment:
We need to be able to have specify Y values within one drive field period with the first and last value exactly at the start and end end of a DF cycle (since we might not know what happens in between patches). The values in between could be equidistantly spread out. If we define it that way we can have static focus fields with Y=1,since the one value will be first and last at the same time.

Reply:
Yes I think we have that if `Y` is a multiple of `J`. Then the first and last value will always be on the boundary. I think we should enforce this.

Reply:
Agree if we include 
> one value will be first and last at the same time.

into the specification

Reply:
yes lets do that.

Reply:
give me a ping if you have finished an update on this one. I will look into it.